### PR TITLE
DBLD: Fix docker build for centos6 and debian sid

### PR DESCRIPTION
**Pull request by mitzkia**
- This PR fixes python related docker builds for CentOS-6 and Debian-Sid

### CODE REVIEW

**Comment by mitzkia** [comment]
@kira-syslogng build packages please

**Comment by mitzkia** [comment]
I have tested docker build manually, on my host with: `dbld/rules images`, and it has passed for all supported platforms:
```console
balabit/syslog-ng-devshell         latest              6742a162530e        13 minutes ago      1.85GB
balabit/syslog-ng-kira             latest              6158139c744e        15 minutes ago      1.38GB
balabit/syslog-ng-ubuntu-focal     latest              510e858d9efd        15 minutes ago      1.21GB
balabit/syslog-ng-ubuntu-bionic    latest              826cac0b1da7        18 minutes ago      1.3GB
balabit/syslog-ng-ubuntu-xenial    latest              bb4d82b983c0        21 minutes ago      816MB
balabit/syslog-ng-debian-sid       latest              a85acd2b1422        24 minutes ago      859MB
balabit/syslog-ng-debian-buster    latest              1c140ef4c23e        26 minutes ago      899MB
balabit/syslog-ng-debian-stretch   latest              5a32f48cbfa2        28 minutes ago      794MB
balabit/syslog-ng-fedora-32        latest              70fa1294fa53        30 minutes ago      1.34GB
balabit/syslog-ng-centos-7         latest              e540bf5228f3        34 minutes ago      1.59GB
balabit/syslog-ng-centos-6         latest              4fad8f305d42        38 minutes ago      1.14GB
```

**Comment by kira-syslogng** [comment]
Package Build SUCCESS

Packages are available at https://ci.syslog-ng.com/jenkins/job/kira-dbld-pull-requests/705/artifact/

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: dbld/builddeps [hunk 1]
```diff
@@ -187,9 +187,18 @@ function install_pip_packages {
...
-        filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs $python_executable -m pip install --ignore-installed -U
+        case "${OS_PLATFORM}" in
+            centos-6)
+                pip install --upgrade pip==9.0.3
+                pip install --upgrade setuptools
+                filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs pip install --ignore-installed -U
+                ;;
+            *)
+                $python_executable -m pip install --upgrade pip
+                $python_executable -m pip install --upgrade setuptools
```
Comment (alltilla) [inline dbld/builddeps:198]: What if we had $pip_command instead $python_executable, so we can have: python2 -m pip, python3 -m pip, python -m pip, pip?

The only challenge would be the devshell, which would have multiple, but space won't be as a good delimiter, as it is now.

After that only the version forcing would be the difference, to which I don't have an idea yet.

**Comment by mitzkia** [comment]
@alltilla Thanks for the notes.
The Jenkinsfile would be a good idea, but we did not use it for now.
If there is not a better idea for the introduced `case - esac`, I would keep this variant.
Maybe we can drop centos-6 support soon, than we can switch back to the previous version :)

**Comment by mitzkia** [comment]
@kira-syslogng build packages please

**Comment by kira-syslogng** [comment]
Package Build FAILURE

**Comment by mitzkia** [comment]
@kira-syslogng build packages please

**Comment by kira-syslogng** [comment]
Package Build SUCCESS

Packages are available at https://ci.syslog-ng.com/jenkins/job/kira-dbld-pull-requests/707/artifact/

**Pull request merged**